### PR TITLE
Send and receive emails in py3 using GAE legacy libraries

### PR DESCRIPTION
**Pull request by jrobbins**
In this PR:
* Move sendemail.py and unit tests from py2/ to framework/
* Update dispatch.yaml to route requests to the default service instead of app-py3
* Stop using service app-py3 and make our py3 code the default service
* Delete everything in py2 and the associated scripts in package.json

### CODE REVIEW

File: dispatch.yaml [hunk 1]
```diff
@@ -14,11 +14,5 @@ dispatch:
...
 
-- url: "*/tasks/outbound-email"
-  service: default
-
-- url: "*/py2"
-  service: default
-
 - url: "*/*"
-  service: app-py3
+  service: default
```
Comment (KyleJu) [inline dispatch.yaml:18]: Remove the comments from line 3 to line 8

File: requirements.txt [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
 funcsigs==1.0.2
 google-python-cloud-debugger==2.18
 google-cloud-tasks==2.7.0
 google-cloud-ndb==1.11.1
 google-cloud-logging==3.0.0
-google-auth==1.24.0
+google-auth==1.31.0
 requests==2.27.1
 
 # TODO(jrobbins): Update to a 2.x version of flask, which may require source
```
Comment (KyleJu) [inline requirements.txt:16]: Maybe we can clean up the Py2 dependencies in a follow-up PR

File: scripts/start_server.sh [hunk 3]
```diff
@@ -28,7 +28,7 @@ gunicorn --bind :8080 --workers 4 main:app
 #  $BASEDIR/../dispatch.yaml \
 #  $BASEDIR/../notifier.yaml \
 #  $BASEDIR/../dev-default.yaml \
```
Comment (KyleJu) [inline scripts/start_server.sh:30]: is `dev-default.yaml` still needed? We can delete the file at this point.

**Review by KyleJu — APPROVED** [review]

File: dispatch.yaml (hunk 1 above)
Comment (jrobbins) [reply dispatch.yaml:14]: I shortened the comments, but I still wanted something in there to explain that inbound email goes to the default service.

File: requirements.txt (hunk 2 above)
Comment (jrobbins) [reply requirements.txt:16]: Yep, I'll try doing that after the next deployment.

File: scripts/start_server.sh (hunk 3 above)
Comment (jrobbins) [reply scripts/start_server.sh]: Good point.  Gone.

**Pull request merged**